docs/src/main/asciidoc/spring-cloud-config.adoc

```diff
@@ -1158,6 +1173,7 @@ AWS Secrets Manager API credentials are determined using link:https://docs.aws.a
 [NOTE]
 ====
 - When no application is specified `application` is the default, and when no profile is specified `default` is used.
+- Both `label` and `defaultLabel` properties are ignored, when `ignoreLabel` one is set to `true`.
```

Comment:
I think you should remove the word `one`

Reply:
Done.